install/write-conf

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+new_user="$1"
+config_file="/usr/local/etc/wp-mass-admin/wpma.conf"
+tmp_file="/tmp/wpma.scratch"
+
+rm -f $tmp_file
+touch $tmp_file
+
+while read line
+do
+    echo $line | grep "userNames" >> /dev/null
+    if [ $? != 0 ]
+    then
+	echo $line >> $tmp_file
+    else
+	userNames="$(echo $line | sed -e 's/userNames=//g' | tr -d '"')"
+	echo -n "userNames=\"" >> $tmp_file
+	for user in $userNames
+	do
+		echo -n "$user " >> $tmp_file
+	done
+	echo -n "$new_user" >> $tmp_file
+	echo "\"" >> $tmp_file
+    fi
+done < $config_file
+
+cat $tmp_file > $config_file
+rm $tmp_file
+
```

Comment:
Same question about this script as with the other write-conf one. Couldn't it be replaced with:

```sed
sed -r "s/userNames=\"(.*)\"/userNames=\"\1 $new_user\"/""
```